client/src/components/MapView/LayerControl.tsx

```diff
@@ -24,10 +24,11 @@ class LayerControl implements IControl {
         this.container.removeChild(this.container.firstChild);
       }
 
-      const title = document.createElement('h6');
+      const title = document.createElement('h4');
       title.textContent = 'Debug layers';
       this.container.appendChild(title);
 
+      const groups: Map<string, HTMLDivElement> = new Map<string, HTMLDivElement>();
```

Comment:
Actually, if you swap Map for Record here, you get the same result (Record is stricter when it comes to types), but you don't have to alias the (Maplibre)Map import.

Reply:
Changed in https://github.com/opentripplanner/OpenTripPlanner/pull/6114/commits/65272ba55b47cc26d807ae992176ff6cffc13736